packages/schematics/integration/integration-test-bed/integration-test-bed.ts

```diff
@@ -0,0 +1,21 @@
+import { RunResult } from '../nixt-facade';
+
+export interface IntegrationTestBed {
```

Comment:
I wanted to hide the constructor

Reply:
Ah, I see.
There is no need to change it right now.

I see three benefits of merging the interface with the implementation:

- Easier naming (no `Impl`-Suffix)
- JS-Doc-Comments are easier to discover.
- If another need to configure the test-directory it can easily be done by using the constructor

An `interface` definitely makes sense if there are multiple implementations.